bin/index.js

```diff
@@ -20,6 +20,7 @@ const program = require('commander')
     .option('-l, --lang [value]', 'specify language of page content')
     .option('-t, --toc [range]', 'specify min and max level of table of contents', range)
     .option('-n, --noToc', 'do not render table of contents')
+    .option('-g, --github [value]', 'add link to Github page')
```

Comment:
Don't forget to add to README as well.

And maybe `username/reponame` makes more sense as notation than `https://github.com/username/reponame`?